package_additions.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Create archive with docker additions for sprint (IDE and IRC).
+docker save briangilbert/cloud9-alpine:latest linuxserver/thelounge:latest | xz -z -9 > ${REPO_DIR}/docker_additions.tar.xz
```

Comment:
Is the Docker build of this image available and checked in? If so, maybe add a comment with the link to it?

Could we change the name of this script to drupal_sprint_package_additions.sh?